### PR TITLE
CMR-7877 Fixing bug when FreeAndOpenData false the data is not transl…

### PR DESCRIPTION
**Pull request by eereiter**
…ated.

### CODE REVIEW

File: umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj [hunk 1]
```diff
@@ -210,7 +210,18 @@
                        </DIF>"
           result (parse/parse-dif10-xml actual-data options)
           umm-result (:UseConstraints result)]
-      (is (= true (boolean (:FreeAndOpenData umm-result))))))
+      (is (= true (:FreeAndOpenData umm-result)))))
```
Comment (jaybarra) [inline umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj:213]: can use `true?`

File: umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj [hunk 2]
```diff
@@ -210,7 +210,18 @@
...
+  (testing "dif10 use constraints Free_And_Open_Data test false"
+    (let [actual-data "<DIF>
+                         <Use_Constraints>
+                           <Description>Description</Description>
+                           <Free_And_Open_Data>false</Free_And_Open_Data>
+                         </Use_Constraints>
+                       </DIF>"
+          result (parse/parse-dif10-xml actual-data options)
+          umm-result (:UseConstraints result)]
+      (is (= false (:FreeAndOpenData umm-result)))))
```
Comment (jaybarra) [inline umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj:224]: can use `false?`

**Review by jaybarra — APPROVED** [review]

**Review by ygliuvt — APPROVED** [review]

File: umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj (hunk 2 above)
Comment (eereiter) [reply umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj:224]: yes, I can. I have updated the text.

File: umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj (hunk 1 above)
Comment (eereiter) [reply umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif10.clj:213]: yes I can. I have updated the text.

**Pull request merged**